### PR TITLE
Social graph Next

### PR DESCRIPTION
**Pull request by jamesgorrie**
Get's the TwitterCard and OpenGraph data working.
Next up I'll deprecate the old nunjucks ones along with the analytics, let's make sure we're happy with this first.

screen shot 2018-02-07 at 12 54 33

### CODE REVIEW

**Comment by jamesgorrie** [comment]
@wellcometrust/experience-dev out of WIP, and ready for prime time.

**Comment by jamesgorrie** [comment]
content and events domain model

File: common/views/components/DefaultPageLayout/DefaultPageLayout.js [hunk 1]
```diff
@@ -45,13 +110,13 @@ type AnalyticsProps = {|
...
 // `dimension5` is a test dimension. it's `dimension1` on v2
-const Analytics = ({
+export const Analytics = ({
   category,
   seriesUrl,
   positionInSeries,
   contentType
 }: AnalyticsProps) => ([
-  <style key='analytics-1'  dangerouslySetInnerHTML={{ __html: `.async-hide .header__nav{ opacity: 0 !important}`}} />,
+  <style  key='analytics-1' dangerouslySetInnerHTML={{ __html: `.async-hide .header__nav{ opacity: 0 !important}` }} />,
```
Comment (davidpmccormick) [inline common/views/components/DefaultPageLayout/DefaultPageLayout.js]: Extra space before `key`?

File: common/views/components/DefaultPageLayout/DefaultPageLayout.js [hunk 2]
```diff
@@ -82,54 +147,84 @@ const Analytics = ({
...
       window.WC = {
         featuresCohort: ${JSON.stringify(featuresCohort)},
         featureFlags: ${JSON.stringify(featureFlags)}
       }
     `}} />
-      {canonicalUri && <link rel="canonical" href={canonicalUri} />}
+      {url && <link rel='canonical' href={url} />}
     </Head>
 
     <div className={isPreview ? 'is-preview' : undefined}>
```
Comment (davidpmccormick) [inline common/views/components/DefaultPageLayout/DefaultPageLayout.js]: Is this ternary required, or would JSX just leave out the class altogether with a falsy `className={isPreview}`?

File: common/views/components/DefaultPageLayout/DefaultPageLayout.js (hunk 1 above)
Comment (jamesgorrie) [reply common/views/components/DefaultPageLayout/DefaultPageLayout.js]: It's to keep it neat with the style below it.

File: common/views/components/DefaultPageLayout/DefaultPageLayout.js (hunk 2 above)
Comment (jamesgorrie) [reply common/views/components/DefaultPageLayout/DefaultPageLayout.js]: Yea, but how would it know to use the string `is-preview`?

We could create something that takes an `{[string]: boolean}` but haven't found many cases for it it.

e.g.
```jsx
<div className={classnames({ 'is-preview' : isPreview })}></div>
```

Comment (davidpmccormick) [reply common/views/components/DefaultPageLayout/DefaultPageLayout.js]: Of course – I'm being slow.

File: common/views/components/DefaultPageLayout/DefaultPageLayout.js (hunk 1 above)
Comment (davidpmccormick) [reply common/views/components/DefaultPageLayout/DefaultPageLayout.js]: Gross!

**Pull request merged**